common/components/constants/LinkConstants.js

```diff
@@ -5,62 +5,63 @@ import type { Dictionary, KeyValuePair } from "../types/Generics.jsx";
 import _ from "lodash";
 import stringHelper from "../utils/string.js";
 
-
 export type LinkSourceDisplayConfig = {|
   +sourceUrlPattern: ?RegExp,
   +sourceDisplayName: ?string,
   +sourceTypeDisplayName: string,
   +iconClass: string,
 |};
 
+const httpsWwwPrefix = "^https?://w*.?";
```

Comment:
```suggestion
const httpsWwwPrefix = "^https?://\\w*\\.?";
```
In looking at our link urls, I notice that we have some that don't have the www subdomain, like 'drive.google.com' and  'join.slack.com'